src/test/java/de/rabea/game/GameTypeTest.java

```diff
@@ -12,29 +14,83 @@
 
     FakeUserInterface fakeUserInterface;
     FakeComputerPlayer fakeComputerPlayer;
-    GameType gameType;
     GameManager gameManager;
+    Console fakeConsole;
 
     @Before
     public void setup() {
         fakeUserInterface = new FakeUserInterface();
         fakeComputerPlayer = new FakeComputerPlayer(O);
         gameManager = new GameManager(fakeUserInterface);
-        gameType = new GameType(fakeUserInterface, gameManager);
+        fakeConsole = new FakeConsole();
     }
 
     @Test
-    public void itSetsUpTheFirstGameAsAHvHGame() {
-        fakeUserInterface.provideConsoleInput("2");
-        Game game = gameType.createGame();
-        assertTrue(game != null);
+    public void createsAHumanVsHumanGame() {
+        GameType gameType = new GameType(new UIReturningHvH(fakeConsole), gameManager);
+        assertTrue(gameType.createGame() != null);
```

Comment:
@jsuchy I can't think of a good way to test this. Could you give me a hint? 

The only two things I can think of are to 
- create a "getPlayer()" method in Game and then assert for the type of player that is created
- make the createPlayer() and createOpponent() methods public and then test for the type of player that is created

But I don't think either options are very good...